### PR TITLE
Record the time taken to perform test

### PR DESCRIPTION
**Pull request by shruacha1234**
Signed-off-by: Shruthi <Shruthi.Acharya@ibm.com>

### CODE REVIEW

File: tests/scripts/common_functions.sh [hunk 1]
```diff
@@ -88,13 +102,30 @@ function setup() {
 	esac
 }
 
+# Check if the prometheus is already deployed , if not invoke the script to deploy prometheus on minikube
+function setup_prometheus() {
```
Comment (chandrams) [inline tests/scripts/common_functions.sh:106]: I see another PR for prometheus setup, should this be removed from this PR?

**Review by chandrams — COMMENTED** [review]
I see another PR for prometheus setup, should this be removed from this PR?

**Review by chandrams — APPROVED** [review]
lgtm

**Review by dinogun — APPROVED** [review]
lgtm

**Pull request merged**